version_test.go

```diff
@@ -0,0 +1,40 @@
+package main_test
+
+import (
+	"bytes"
+	"os/exec"
+	"testing"
+
+	"github.com/onsi/gomega/gexec"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+)
+
+func testVersion(t *testing.T, context spec.G, it spec.S) {
+	var (
+		withT      = NewWithT(t)
+		Expect     = withT.Expect
+		Eventually = withT.Eventually
+
+		buffer *bytes.Buffer
+	)
+
+	it.Before(func() {
+		buffer = bytes.NewBuffer(nil)
+	})
+
+	context("when the format is set to markdown", func() {
+		it("prints out the summary of a buildpack tarball", func() {
```

Comment:
@ForestEckhardt can you change this to something relevant about getting the version?

Reply:
Lol yes sorry about that

Reply:
Should be all set now